src/model/page-config-resolver-test.js

```diff
@@ -177,6 +177,22 @@ describes.realWin('PageConfigResolver', {}, env => {
           .to.equal('pub1:basic');
     });
 
+    it('should allow alternate LD Types', () => {
+      schema['@type'] = ['CreativeWork'];
+      addJsonLd(schema);
+      readyState = 'complete';
+      expect(new PageConfigResolver(gd).check().getProductId())
+          .to.equal('pub1:basic');
+    });
+
+    it('should allow work with multiple LD types', () => {
+      schema['@type'] = ['NewsArticle', 'Person'];
+      addJsonLd(schema);
+      readyState = 'complete';
+      expect(new PageConfigResolver(gd).check().getProductId())
+          .to.equal('pub1:basic');
+    });
+
```

Comment:
could we have a negative test case for a number of types but none of them feature in the Allowed type list.

Reply:
done